plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -292,6 +292,10 @@
      </detector>
   </extension>
   
+  <!-- 
+    cannot use com.google.cloud.tools.eclipse.appengine.onlyForStandardProjects for the trigger
```

Comment:
what file is this in? I couldn't find it.

Reply:
It's defined in this file [further above](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/105ea90a3b14a914222c00b4dcce26b7f6a35272/plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml#L143)